### PR TITLE
Enable collecting Appliance and VCH log

### PR DESCRIPTION
**Pull request by luwang-vmware**
Only collect appliance log when case is failure

### CODE REVIEW

File: tests/resources/OVA-Util.robot [hunk 1]
```diff
@@ -223,11 +223,12 @@ Copy Support Bundle
     ${output}=  Gather Support Bundle
     Should Contain  ${output}  Created log bundle
     ${file}=  Get Support Bundle File  ${output}
+    Log To Console  bundle file is ${file}
 
     Close connection
 
     # copy log bundle
-    ${output}=  Run command and Return output  sshpass -p ${OVA_PASSWORD_ROOT} scp -o StrictHostKeyChecking\=no -o UserKnownHostsFile=/dev/null ${OVA_USERNAME_ROOT}@${ova-ip}:${file} .
+    ${output}=  Run command and Return output  sshpass -p ${OVA_PASSWORD_ROOT} scp -o StrictHostKeyChecking\=no -o UserKnownHostsFile=/dev/null ${OVA_USERNAME_ROOT}@${ova-ip}:${file} ${OUTPUT DIR}
```
Comment (DanielXiao) [inline tests/resources/OVA-Util.robot]: Where is OUTPUT DIR defined?

File: tests/manual-test-cases/Group5-Interoperability-Tests/5-03-Multiple-Datacenter.robot [hunk 2]
```diff
@@ -17,6 +17,7 @@ Documentation  Test 5-03 - Multiple Datacenters
 Resource  ../../resources/Util.robot
 Suite Setup  Nimbus Suite Setup  Multiple Datacenter Setup
 Suite Teardown  Run Keyword And Ignore Error  Nimbus Cleanup  ${list}
+Test Teardown  Run Keyword If  '${TEST STATUS}' != 'PASS'  Collect Appliance and VCH Logs  ${VCH-NAME}
```
Comment (DanielXiao) [inline tests/manual-test-cases/Group5-Interoperability-Tests/5-03-Multiple-Datacenter.robot]: Why do we move 5-03 and 5-05 back? I think we can explain it with a separated commit in this PR.

Comment (luwang-vmware) [reply tests/manual-test-cases/Group5-Interoperability-Tests/5-03-Multiple-Datacenter.robot]: Ok, will separate them.

File: tests/resources/OVA-Util.robot (hunk 1 above)
Comment (luwang-vmware) [reply tests/resources/OVA-Util.robot]: OUTPUT DIR is defined by ROBOT itself.
http://robotframework.org/robotframework/latest/RobotFrameworkUserGuide.html

**Pull request merged**